docs-yaml/src/extension.ts

```diff
@@ -60,6 +62,12 @@ async function removeTocSchemaFromConfigAtScope(value: string, scope: Configurat
     await workspace.getConfiguration().update(YAML_SCHEMA_CONFIG_NAME_OF_VSCODE_YAML_EXTENSION, newValue, scope);
 }
 
+// retrieve mapping file data and store the data in variable
+export async function loadSchemaConfig() {
+    const getResult = await WebRequest.get('https://static.docs.com/ui/latest/schemas/schema_config.json');
```

Comment:
Is there a config file for the project that we can pull constants from?

Reply:
There is a constant file with static values that we can use: https://github.com/Microsoft/vscode-docs-authoring/blob/develop/docs-yaml/src/yaml-support/yaml-constant.ts

Do you think we should store the mappingFile constant here?

Reply:
yeah, that looks like a good spot.